libs/components/forms/testing/src/form-error/form-errors-harness.ts

```diff
@@ -35,84 +24,73 @@ export class SkyFormErrorsHarness extends SkyComponentHarness {
   }
 
   /*
-   * Gets total number of errors.
-   */
-  public async getNumberOfErrors(): Promise<number> {
-    return (await this.#getFormError()).length;
-  }
-
-  /*
-   * Gets if the required error has triggered.
+   * Whether the required field is set to an empty value.
    */
   public async hasRequiredError(): Promise<boolean> {
-    return (await this.#getFormErrorsClasses()).includes(
-      'sky-form-error-required',
-    );
+    return this.#hasErrorOfType('required');
   }
 
   /*
-   * Gets if the maximum length error has triggered.
+   * Whether the field has more characters than is allowed.
```

Comment:
```suggestion
   * Whether the field has more characters than allowed.
```
I think this reads slightly better without "is." No worries if you disagree though.